pkgs/applications/misc/electrum/default.nix

```diff
@@ -22,12 +23,12 @@ let
 in
 python3.pkgs.buildPythonApplication rec {
   pname = "electrum";
-  version = "4.6.0";
+  version = "4.6.2";
   format = "setuptools";
```

Comment:
Would be nice if we can update this to pyproject = true

Reply:
Done in a separate commit "electrum: init pyproject".